packages/server/src/templates/templates.service.tsx

```diff
@@ -11,11 +11,17 @@ import createEmotionCache from './projects/common/createEmotionCache';
 export class TemplatesService {
   private readonly logger = new Logger(TemplatesService.name);
 
+  /**
+   * Get the template based on the given name.
+   *
+   * Will return null if the template is not found.
+   */
   async getTemplate(templateName: string, props: any): Promise<any> {
-    const TemplateFile = await import('./projects/' + templateName).catch(err => {
-      this.logger.log(err);
-    })
-    const SelectedTemplate = TemplateFile.default
+    // Get the template file
+    const SelectedTemplate = await this.importTemplate(templateName);
+    if (SelectedTemplate === null) {
+      return null;
```

Comment:
Instead of returning null, throw an error here, and then this function can return a string.